consul/service.sls

```diff
@@ -0,0 +1,12 @@
+include:
+  - .install
+  - .config
+
+start_consul_service:
+  service.running:
+    - name: consul
+    - enable: True
+    - require:
+      - configure_consul_service
+    - watch:
+      - configure_consul_service
```

Comment:
The service shouldn't directly require or watch anything. The service state should be included into other states and have the steps in those states should use the `require_in` or `watch_in` attributes to interface with the service.